python/sglang/srt/server_args.py

```diff
@@ -1436,6 +1467,46 @@ def _handle_other_validations(self):
 
     @staticmethod
     def add_cli_args(parser: argparse.ArgumentParser):
+
+        # Ktransformer server args
+        parser.add_argument(
+            "--amx-weight-path",
+            type=str,
+            help="The path of the quantized expert weights for amx kernel. A local folder.",
+        )
+        parser.add_argument(
+            "--amx-method",
+            type=str,
+            default="AMXINT4",
+            help="Quantization formats for CPU execution.",
+        )
+        parser.add_argument(
+            "--cpu-embed",
+            action="store_true",
+            help="Enable moving embed_token to cpu",
+        )
+        parser.add_argument(
+            "--cpuinfer",
+            type=int,
+            help="The number of CPUInfer threads.",
+        )
+        parser.add_argument(
+            "--subpool-count",
+            type=int,
+            default=2,
+            help="The number of NUMA nodes.",
+        )
+        parser.add_argument(
+            "--num-gpu-experts",
+            type=int,
+            help="The number of GPU experts.",
+        )
+        parser.add_argument(
+            "--enable-defer",
+            action="store_true",
+            help="Enable Expert Deferral.",
+        )
```

Comment:
qq: is all these only used for ktransformer, or also useful for other cases? if the former maybe we should do a prefix, o/w users may get confused

EDIT: seems some are good, but some e.g. "subpool-count" may be confusing if w/o context